rule/expr.go

```diff
@@ -461,6 +581,111 @@ func (v *Value) GT(other *Value) bool {
 	return false
 }
 
+// GTE reports whether v is greater or equal than other.
+func (v *Value) GTE(other *Value) bool {
+	switch v.Type {
+	case "bool":
+		v1, _ := strconv.ParseBool(v.Data)
```

Comment:
:tulip: We're doing a whole load of this  `strconv.Parse...` stuff, in the 0.7 version we moved this to common functions.  I don't think it's worth doing a lot of re-factoring here, though, so I stuck a :tulip: on it ;-)

Reply:
Yeah I opted for the Quick&Dirty here because I know that this will be removed in the 0.7.